specification/cdn/resource-manager/Microsoft.Cdn/stable/2019-04-15/cdn.json

```diff
@@ -3399,15 +3399,15 @@
           ]
         },
         "sourcePattern": {
-          "description": "define a request URI pattern that identifies the type of requests that may be rewritten. If value is blank, all strings are matched.",
+          "description": "define a request URI pattern that identifies the type of requests that may be rewritten. Currently, source pattern uses a prefix-based match. To match all URL paths, use “/” as the source pattern value. To match only the root directory and re-write this path, use the origin path field",
```

Comment:
@hytao could you confirm “/” is valid? It seems this double quote is not valid half-width symbol. 
If you want to use double quote, please use \" to escape

Reply:
The issue is is fixed now.